nixos/tests/firewall.nix

```diff
@@ -60,7 +60,10 @@ import ./make-test-python.nix ( { pkgs, nftables, ... } : {
     walled.stop_job("${unit}")
     attacker.succeed("curl -v http://walled/ >&2")
 
-    # Check whether activation of a new configuration reloads the firewall.
+    # This sleep seems to mitigate some race condition that prevents the firewall being
+    # reloaded/started as part of the activation of the new config.
+    walled.sleep(1)
```

Comment:
Can we somehow wait or check something? one second might not be enough for the hydra builders

Reply:
I have no idea what to wait for. The [stop_job](https://github.com/NixOS/nixpkgs/blob/edbebf5ce4cba33e777eb038aa9b95f947bef518/nixos/tests/firewall.nix#L60) blocks until the unit is stopped. Even the subsequent curl request on the previously walled resource finishes.
But this means in fact that the D-Bus response to `switch-to-configuration.pl`'s `get_active_units()` contains an outdated state.

Since it's not related to this PR, I reverted the change, and I'd like to leave it as is.